schemas/user.py

```
@@ -1,103 +1,65 @@
     @classmethod
-    def validate_password_complexity(cls, v: Optional[str]):
-        # 비밀번호를 입력하지 않은 경우(None)는 검증 통과 (수정 안 함)
-        if v is None:
-            return v
-
-        if not re.search(r"[a-zA-Z]", v):
-            raise ValueError("비밀번호에는 최소 하나 이상의 영문자가 포함되어야 합니다.")
-
-        if not re.search(r"\d", v):
-            raise ValueError("비밀번호에는 최소 하나 이상의 숫자가 포함되어야 합니다.")
+    def validate_new_password(cls, v: Optional[str]):
```

Ruff (0.14.13)

Missing return type annotation for classmethod `validate_new_password`

(ANN206)